src/Scaffold_Command.php

```diff
@@ -0,0 +1,307 @@
+<?php
+
+use EE\Process;
+use EE\Utils;
+
+class Scaffold_Command extends EE_Command {
+
+	/**
+	 * Generate a README.md for your command.
+	 *
+	 * Creates a README.md with Using, Installing, and Contributing instructions
+	 * based on the composer.json file for your EasyEngine package. Run this command
+	 * at the beginning of your project, and then every time your usage docs
+	 * change.
+	 *
+	 * These command-specific docs are generated based composer.json -> 'extra'
+	 * -> 'commands'. For instance, this package's composer.json includes:
+	 *
+	 * ```
+	 * {
+	 *   "name": "easyengine/scaffold-command",
+	 *    // [...]
+	 *    "extra": {
+	 *        "commands": [
+	 *            "scaffold package-readme"
+	 *        ]
+	 *    }
+	 * }
+	 * ```
+	 *
+	 * ## OPTIONS
+	 *
+	 * <dir>
+	 * : Directory path to an existing package to generate a readme for.
+	 *
+	 * [--force]
+	 * : Overwrite the readme if it already exists.
+	 *
+	 * @subcommand package-readme
+	 */
+	public function package_readme( $args, $assoc_args ) {
+
+		list( $package_dir ) = $args;
+
+		self::check_if_valid_package_dir( $package_dir );
+
+		$composer_obj = json_decode( file_get_contents( $package_dir . '/composer.json' ), true );
+		if ( ! $composer_obj ) {
+			EE::error( 'Invalid composer.json in package directory.' );
+		}
+
+		$force = Utils\get_flag_value( $assoc_args, 'force' );
+
+		$package_root  = dirname( dirname( __FILE__ ) );
+		$template_path = $package_root . '/templates/';
+
+		$bits        = explode( '/', $composer_obj['name'] );
+		$readme_args = array(
+			'package_name'              => $composer_obj['name'],
+			'package_short_name'        => $bits[1],
+			'package_name_border'       => str_pad( '', strlen( $composer_obj['name'] ), '=' ),
+			'package_description'       => isset( $composer_obj['description'] ) ? $composer_obj['description'] : '',
+			'shields'                   => '',
+			'has_commands'              => false,
+			'ee_update_to_instructions' => 'the latest stable release with `ee cli update`',
+			'show_powered_by'           => isset( $composer_obj['extra']['readme']['show_powered_by'] ) ? (bool) $composer_obj['extra']['readme']['show_powered_by'] : true,
+		);
+
+		if ( isset( $composer_obj['extra']['readme']['shields'] ) ) {
+			$readme_args['shields'] = implode( ' ', $composer_obj['extra']['readme']['shields'] );
+		} else {
+			$shields = array();
+			if ( file_exists( $package_dir . '/.travis.yml' ) ) {
+				$shields[] = "[![Build Status](https://travis-ci.org/{$readme_args['package_name']}.svg?branch=master)](https://travis-ci.org/{$readme_args['package_name']})";
```

Comment:
Need the url with `EasyEngine` written to show correct badge. This will give in all small letters. Generating non-working badges.

Example:

* Working Badge: [![Build Status](https://travis-ci.org/EasyEngine/site-command.svg?branch=master)](https://travis-ci.org/easyengine/site-command) 
`[![Build Status](https://travis-ci.org/EasyEngine/site-command.svg?branch=master)](https://travis-ci.org/easyengine/site-command)`

* Badge that this code will generate: [![Build Status](https://travis-ci.org/easyengine/site-command.svg?branch=master)](https://travis-ci.org/easyengine/site-command)
`[![Build Status](https://travis-ci.org/easyengine/site-command.svg?branch=master)](https://travis-ci.org/easyengine/site-command)`